### PR TITLE
Merchant template

### PR DESCRIPTION
**Pull request by TheShadowOfHassen**

### CODE REVIEW

File: 0-ad-history-mod-package/simulation/templates/template_unit_ship_merchant.xml [hunk 1]
```diff
@@ -16,6 +16,9 @@
   <Identity>
     <GenericName>Merchantman</GenericName>
     <SelectionGroupName>template_unit_ship_merchant</SelectionGroupName>
+    <History>
+        Trade on the high seas was a dangerous but lucrative activity. Merchants braved, rough seas, storms, and pirates, while traversing sea lanes that spanned empires. On the way the merchants and sailors did not only buy and sell goods, but spread news, customs, and ideas, enriching the societies they encountered.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/template_unit_ship_merchant.xml:20]:
```suggestion
        Trade on the high seas was a dangerous but lucrative activity. Merchants braved, rough seas, storms, and pirates, while traversing sea lanes that spanned empires. On the way the merchants and sailors did not only buy and sell goods but spread news, customs, and ideas, enriching the societies they encountered.
```

**Review by indoptogopt — CHANGES_REQUESTED** [review]

**Comment by TheShadowOfHassen** [comment]
Is this OK to merge?

**Comment by indoptogopt** [comment]
Yes. What do you think of using this for all merchants?(including the ones we've already written)

**Comment by TheShadowOfHassen** [comment]
For all merchant ships? It could be. I mean, it's a template so it's already for everything that we don't override with a custom template

**Pull request merged**